install-usercss/install-usercss.js

```diff
@@ -64,35 +48,6 @@
     }
   }, 200);
 
-  getTab(tabId).then(tab => (tabUrl = tab.url));
-  chrome.tabs.onUpdated.addListener((id, {url}) => {
-    if (id === tabId && url && url !== tabUrl) {
-      closeCurrentTab();
-    }
-  });
-  // close the tab in case the port didn't report onDisconnect
```

Comment:
I have no idea and I never remove things that ain't broke in a PR that's not about this particular thing.

Reply:
Then I think we have to move this to `PortDownloader`. I think there is a bug that `port.onDisconnected` doesn't fire while the source tab navigates to a different URL.

Reply:
I wonder if it is because Firefox keeps the document in the memory in this case. We may have to investigate this situation. We can delay this to a new issue.

Reply:
#864